### ISSUE TITLE
Default article footer anchor text does not meet minimum contrast criteria

### ISSUE THREAD

**Issue by jeremyfelt** [washingtonstateuniversity/WSUWP-spine-parent-theme#372]
See: https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html

**Comment by jeremyfelt** [comment]
Fixed in #373
